fdbserver/DDRelocationQueue.actor.cpp

```diff
@@ -1875,8 +1887,9 @@ ACTOR Future<Void> dataDistributionRelocator(DDQueue* self,
 						dataTransferComplete.send(rd);
 					}
 
+					int nonOverlappingCount = nonOverlappedSrcCount(rd.completeSources, destIds);
```

Comment:
Can you add a comment here about what it means to use `rd.completeSources` for `srcIds`, in particular, during merges?

Reply:
added